app/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
I'm guessing this `ids.xml` file was created as part of troubleshooting the settingsButton issue, but once the typo is corrected, you won't need it.